syntax = "proto3";

package command;

service CommandService {
  rpc ExecuteCommand (CommandRequest) returns (CommandResponse) {}
}

message CommandRequest {
  string command = 1;
}

message CommandResponse {
  string result = 1;
}